index.bs

```diff
@@ -673,7 +673,7 @@ run these steps:
 		of the {{document}}, and update |container| to be
 		the <a>browsing context container</a> of |container|.
 	2. Map |intersectionRect| to the coordinate space of |container|.
-	3. If |container| is scrollable, apply the {{IntersectionObserver}}’s
+	3. If |container| is a <a href="https://www.w3.org/TR/css-overflow/#scroll-container">scroll-container</a>, apply the {{IntersectionObserver}}’s
```

Comment:
Pretty sure there's a better way to do this properly with bikeshed. `<a>scroll container</a>` might be enough?

Reply:
That worked. Thanks for the suggestion!